config/initializers/rollbar.rb

```diff
@@ -1,13 +1,12 @@
+# frozen_string_literal: true
 Rollbar.configure do |config|
   # Without configuration, Rollbar is enabled in all environments.
   # To disable in specific environments, set config.enabled=false.
 
   config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
 
   # Here we'll disable in 'test':
-  if Rails.env.test?
-    config.enabled = false
-  end
+  config.enabled = false if Rails.env.test?
```

Comment:
This is copipaste edited by rubocop, I'll fix it, but as is it easy to understand and may be better sometimes.